Makefile

```diff
@@ -153,6 +154,7 @@ help:
 	@echo "-------------------------------------------------------------------------------"
 
 build: python_check
+	$(ECHO)$(PYTHON) -m pip install -U -r requirements-dev.txt
```

Comment:
Install pip development requirements during installation. Eventually this should be removed or all dependencies installed with pip or similar, but for type-checking and python3 porting it's worth keeping IMO.